jo-rdan/Free-Mentor
# #167921952 Finishes User can signup endpoint

#### What does this PR do?
It creates an endpoint for signing up
#### Description of Task to be completed?
- Write unit tests for signup
- Add validations to fields entered by the user
#### How should this be manually tested?
- clone this repo
- Run the server by typing **npm start** 
- Open POSTMAN and create a new POST request and use this endpoint **localhost:5000/api/v1/auth/signup**, enter all the required fields in the wrong way and it should return a relevant error message
- If all fields are correct it should return a **201** status code indicating that the user is created successfully.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#167921952](https://www.pivotaltracker.com/story/show/167921952)
